index.yaml

```diff
@@ -2600,7 +2602,7 @@ entries:
     description: Collects several related Helm charts, Grafana dashboards, and Prometheus
       rules combined with documentation and scripts to provide easy to operate end-to-end
       Kubernetes cluster monitoring with Prometheus using the Prometheus Operator.
-    digest: 41b2bff949e5b318196f9ca028b1cf97e8739d535dc0758ba2be6da89960d947
+    digest: 88c5fbe910af3fe218bbc14e92b357fe24918ba60ab6b77759a6dcc8d3303551
```

Comment:
same here, not sure whats going on, might try rebasing and re-generating

Reply:
Steps followed: 
1. `git fetch upstream`
2. `git pull upstream dev-v2.5 --rebase`
3. noted that `Current branch backport-istio is up to date.`
4. `make charts`
5. `git status`
6. noted that `nothing to commit, working tree clean` 
7. `helm repo index --url assets assets`
8. `mv assets/index.yaml .`
9. `chmod +x index.yaml`
10. `git diff` and noted that the only changes that occured to the index.yaml were the `created` timestamps 

Generally I don't run `make charts` without setting the `export PACKAGE=<whatever chart package I am working on>` unless there was an issue with the checks passing for the PR, which did happen with this PR a few times. When I ran `make charts` it is possible that the digest changed on some of the charts, even if the content didn't change.

Reply:
Ensuring that the digest did not change is one of the reasons why I still think we should prioritize https://github.com/rancher/charts-build-scripts/issues/56 to ensure that the inner contents of TGZ files have not been modified. 

Right now, we make an assumption that, the assets/ directory did not change [if nothing changed for that chart in charts/](https://github.com/rancher/charts-build-scripts/blob/deb66c8dd574ddd1f3b58271fec97b7b141588d0/main.go#L193).

cc: @cbron

Reply:
The short-term fix here to avoid changes in future would be to delete the charts that need to change from the index.yaml instead of regenerating entirely.